### PR TITLE
Can we migrate the docs from minerealm to the docs site?

### PR DESCRIPTION
**Pull request by murdified**
This is mostly a a cut/paste from minerealm with a little bit of formatting updated for markdown. I know it's not completely up to date with the actual code, but it's at least as correct as the minerealm page and will make it easier to contribute updates to the documentation.

### CODE REVIEW

**Comment by murdified** [comment]
Sorry, I know you are not seeking PRs or contributions right now, but I offer these formatted docs in case it can be of any use.

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-mkdocs==1.0.4
\ No newline at end of file
+mkdocs==1.2.1
```
Comment (Intelli) [inline requirements.txt:1]: MkDocs 1.0.4 is the latest version supported on the site at the moment, which is why the deploy failed.

We may be able to just change this back to `mkdocs` now though, which should target the latest available version.

**Review by Intelli — CHANGES_REQUESTED** [review]
There are a few unnecessary formatting changes, such as blank lines added to the README file, and closing hashes being added to headers.

File: requirements.txt (hunk 1 above)
Comment (murdified) [reply requirements.txt]: Done

**Pull request merged**